js/grid.js

```diff
@@ -67,7 +67,8 @@
       addEventListener("resize", this);
 
       gNewTab.sendToBrowser("NewTab:InitializeGrid");
-      gNewTab.registerListener("NewTab:UpdateLinks", this.refresh.bind(this));
+      gNewTab.registerListener("NewTab:InitializeLinks",
+                               this.refresh.bind(this));
```

Comment:
Nit, whitespace. jscs formatter will fix this, but as a general rule, just add two spaces.